Logos compilation error if multiline comment has //

Logos fails to compile this code (using latest commit from master branch):
```
%ctor{
    /* http://www.google.com */
}
```
Error log:
```
foxfortmobile@Foxfort-MacBook-Pro dummy % make clean do all messages=1
(printf "\e[0;3%im==> \e[1;39m%s…\e[m\n" 6 "Cleaning"); set -o pipefail; (rm -rf "/Users/foxfortmobile/Documents/workspace/dummy/.theos/obj")
==> Cleaning…
set -o pipefail; (rm "/Users/foxfortmobile/Documents/workspace/dummy/.theos/build_session")
set -o pipefail; (touch "/Users/foxfortmobile/Documents/workspace/dummy/.theos/build_session")
set -o pipefail; (rm -rf "/Users/foxfortmobile/Documents/workspace/dummy/.theos/_")
> Making clean in dummysettings…
(printf "\e[0;3%im==> \e[1;39m%s…\e[m\n" 6 "Cleaning"); set -o pipefail; (rm -rf "/Users/foxfortmobile/Documents/workspace/dummy/.theos/obj")
==> Cleaning…
set -o pipefail; (rm "/Users/foxfortmobile/Documents/workspace/dummy/.theos/build_session")
set -o pipefail; (touch "/Users/foxfortmobile/Documents/workspace/dummy/.theos/build_session")
> Making all for bundle com.foxfort.dummy…
set -o pipefail; (mkdir -p "/Users/foxfortmobile/Documents/workspace/dummy/.theos/obj/debug/com.foxfort.dummy.bundle")
set -o pipefail; (mkdir -p "/Users/foxfortmobile/Documents/workspace/dummy/.theos/obj/debug/com.foxfort.dummy.bundle")
set -o pipefail; (mkdir -p "/Users/foxfortmobile/Documents/workspace/dummy/.theos/obj/debug/com.foxfort.dummy.bundle")
(printf "\e[0;3%im==> \e[1;39m%s…\e[m\n" 5 "Copying resource directories into the bundle wrapper"); set -o pipefail; (for d in Resources; do \
                if [[ -d "$d" ]]; then \
                        rsync -a "$d/" "/Users/foxfortmobile/Documents/workspace/dummy/.theos/obj/debug/com.foxfort.dummy.bundle/" --exclude "_MTN" --exclude ".git" --exclude ".svn" --exclude ".DS_Store" --exclude "._*"; \
                else \
                        printf "\e[0;33m==> \e[1;33mWarning:\e[m %s\n" "Ignoring missing bundle resource directory $d." >&2; \
                fi; \
        done)
==> Copying resource directories into the bundle wrapper…
/Library/Developer/CommandLineTools/usr/bin/make -f Makefile --no-keep-going COLOR=1 \
                internal-bundle-compile \
                _THEOS_CURRENT_TYPE=bundle THEOS_CURRENT_INSTANCE=com.foxfort.dummy _THEOS_CURRENT_OPERATION=compile \
                THEOS_BUILD_DIR="." _THEOS_MAKE_PARALLEL=yes
==> Warning: No files to link - creating a bundle containing only resources
> Making all for tweak dummy…
/Library/Developer/CommandLineTools/usr/bin/make -f Makefile --no-keep-going COLOR=1 \
                internal-library-compile \
                _THEOS_CURRENT_TYPE=tweak THEOS_CURRENT_INSTANCE=dummy _THEOS_CURRENT_OPERATION=compile \
                THEOS_BUILD_DIR="." _THEOS_MAKE_PARALLEL=yes
/Library/Developer/CommandLineTools/usr/bin/make -f Makefile --no-print-directory --no-keep-going internal-tweak-compile _THEOS_CURRENT_TYPE="tweak" THEOS_CURRENT_INSTANCE="dummy" _THEOS_CURRENT_OPERATION="compile" THEOS_BUILD_DIR="." THEOS_CURRENT_ARCH="arm64"
set -o pipefail; (mkdir -p /Users/foxfortmobile/Documents/workspace/dummy/.theos/obj/debug/arm64/)
(printf "\e[0;3%im==> \e[1;39m%s…\e[m\n" 1 "Preprocessing Tweak3.xm"); set -o pipefail; (/Users/foxfortmobile/theos/bin/logos.pl -c warnings=error -c generator=MobileSubstrate    Tweak3.xm > /Users/foxfortmobile/Documents/workspace/dummy/.theos/obj/debug/arm64/Tweak3.xm.mm)
==> Preprocessing Tweak3.xm…
set -o pipefail; (mkdir -p /Users/foxfortmobile/Documents/workspace/dummy/.theos/obj/debug/arm64/)
(printf "\e[0;3%im==> \e[1;39m%s…\e[m\n" 2 "Compiling Tweak3.xm (arm64)"); set -o pipefail; (/Library/Developer/CommandLineTools/usr/bin/clang++ -x objective-c++ -c -I"/Users/foxfortmobile/Documents/workspace/dummy" -I/Users/foxfortmobile/theos/include -I/Users/foxfortmobile/theos/vendor/include -I/Users/foxfortmobile/theos/include/_fallback -include /Users/foxfortmobile/theos/Prefix.pch -MT /Users/foxfortmobile/Documents/workspace/dummy/.theos/obj/debug/arm64/Tweak3.xm.476fcf10.o -MMD -MP -MF "/Users/foxfortmobile/Documents/workspace/dummy/.theos/obj/debug/arm64/Tweak3.xm.476fcf10.Td" -fcolor-diagnostics -DTARGET_IPHONE=1 -O0 -Wall -ggdb -Werror  -isysroot "/Users/foxfortmobile/theos/sdks/iPhoneOS12.0.sdk" -miphoneos-version-min=12.0 -fmodules -fcxx-modules -fmodule-name=dummy -fbuild-session-file=/Users/foxfortmobile/Documents/workspace/dummy/.theos/build_session -fmodules-prune-after=345600 -fmodules-prune-interval=86400 -fmodules-validate-once-per-build-session   -fobjc-arc -DDEBUG -O0  -DTHEOS_INSTANCE_NAME="\"dummy\"" -fmodules -fcxx-modules -fmodule-name=dummy -fbuild-session-file=/Users/foxfortmobile/Documents/workspace/dummy/.theos/build_session -fmodules-prune-after=345600 -fmodules-prune-interval=86400 -fmodules-validate-once-per-build-session  -arch arm64 -F/Users/foxfortmobile/theos/sdks/iPhoneOS12.0.sdk/System/Library/PrivateFrameworks      -stdlib=libc++        /Users/foxfortmobile/Documents/workspace/dummy/.theos/obj/debug/arm64/Tweak3.xm.mm -o /Users/foxfortmobile/Documents/workspace/dummy/.theos/obj/debug/arm64/Tweak3.xm.476fcf10.o)
==> Compiling Tweak3.xm (arm64)…
Tweak3.xm:3:1: error: expected '}'
^
Tweak3.xm:1:129: note: to match this '{'
static __attribute__((constructor)) void _logosLocalCtor_f8b29cb7(int __unused argc, char __unused **argv, char __unused **envp){
                                                                                                                                ^
1 error generated.
make[3]: *** [/Users/foxfortmobile/Documents/workspace/dummy/.theos/obj/debug/arm64/Tweak3.xm.476fcf10.o] Error 1
rm /Users/foxfortmobile/Documents/workspace/dummy/.theos/obj/debug/arm64/Tweak3.xm.mm
make[2]: *** [/Users/foxfortmobile/Documents/workspace/dummy/.theos/obj/debug/arm64/dummy.dylib] Error 2
make[1]: *** [internal-library-all_] Error 2
make: *** [dummy.all.tweak.variables] Error 2
foxfortmobile@Foxfort-MacBook-Pro dummy % 
```

Delete the comment and the compilation succeeds. I have spent quite some time figuring it out and found that it is this commit that causing it:
[ed22e75b10dbee6bea6b000893731822f8bd3874: Fix incorrect matching of //* as /*](https://github.com/theos/logos/commit/ed22e75b10dbee6bea6b000893731822f8bd3874)

If I remove the `//` as below, compilation succeeds:
```
%ctor{
    /* http:www.google.com */
}
```
**Logos should be able to compile irrespective of what I write in commented lines.**

--- a/bin/logos.pl
+++ b/bin/logos.pl
@@ -91,7 +91,8 @@ READLOOP: while(my $line = <FILE>) {
 	# Delete all single-line to-EOL // xxx comments.
 	# This needs to be first so that //*something doesn't match as a /*
 	while($line =~ /\/\//g) {
-		next if fallsBetween($-[0], @quotes);
+		# If // is part of a single-line /**/ comment, skip and handle below
+		next if ($line =~ /^.*?\*\/\s*/ || fallsBetween($-[0], @quotes));
 		$line = $`;
 		redo READLOOP;
 	}

